docs/Project.toml

```diff
@@ -1,3 +1,6 @@
 [deps]
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
 Tensors = "48a634ad-e948-5137-8d70-aa71f2a747f4"
+
+[compat]
+Documenter = "0"
```

Comment:
```suggestion
Documenter = "0"

```

Is this valid?

Reply:
I guess this was from the pre documenter 1.0 era but was added in automatically, but too long ago to remember if there was a reason... But isn't required since we have the Manifest checked in and could be removed.